pyanaconda/kickstart.py

```diff
@@ -80,12 +80,32 @@
 from pykickstart.version import returnClassForVersion
 
 import logging
-log = logging.getLogger("anaconda")
+anaconda_log = logging.getLogger("anaconda")
+script_log = logging.getLogger("anaconda.kickstart.script")
+parsing_log = logging.getLogger("anaconda.kickstart.parsing")
 stderrLog = logging.getLogger("anaconda.stderr")
 storage_log = logging.getLogger("blivet")
 stdoutLog = logging.getLogger("anaconda.stdout")
 from pyanaconda import anaconda_log
 
+# command specififc loggers
+authconfig_log = logging.getLogger("anaconda.kickstart.authconfig")
```

Comment:
Or create the prefix automatically. Something like use class.__name__ or similar. 
Or implement create new "base class" with `log_prefix` variable which will be inherited by these kickstart classes and they just fill the `log_prefix` variable, everything else will be done inside of the "base class".